src/commcare_cloud/ansible/roles/users/meta/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+dependencies:
+  - {role: supervisor, tags: supervisor}
+  - {role: bootstrap-users, tags: bootstrap-users}
```

Comment:
nit: we have the convention of making all files end in a newline

Reply:
Added it here
https://github.com/dimagi/commcare-cloud/pull/1807/commits/ba038c9a3d1715d008c2267567584777ea34238a